CIME/case/case_submit.py

```diff
@@ -51,8 +51,9 @@ def _submit(
 
     # Check if CONTINUE_RUN value makes sense
     # if submitted with a prereq don't do this check
+    testcase = case.get_value("TESTCASE")
     if (
-        job != "case.test"
+        (not testcase or testcase not in ("PET", "NCK"))
```

Comment:
Do you know why these test cases are exceptions? If so, a comment here would be helpful for future maintenance.